### DISCUSSION TITLE
tags_all.py & tags.py

### DISCUSSION THREAD

**Discussion by BRTSG-FOSS** [Bridgetek/IDM2040-7A-EW2022#20] [General]
One code is enough: 
https://github.com/Bridgetek/IDM2040-7A-EW2022/blob/main/LDS_Demo/ui_lds/tags_all.py
https://github.com/Bridgetek/IDM2040-7A-EW2022/blob/main/LDS_Demo/ui_lds/tags.py

**Comment by jackqfzhang** [comment]
thanks for the feedback, updated
